### DISCUSSION TITLE
Class name file name uyuşmazlığı

### DISCUSSION THREAD

**Discussion by BBZKRT** [gusanmaz/BMB212-Algorithms-2223#22] [Q&A] [answered]
Merhaba 
6.ödevde Stack.java adlı file oluşturun demişsiniz .Class ın ismi de mecburi olarak Stack olacak ama biz yeni bir Stack nesnesi tanımladığımızda hata oluşuyor. İsim karışıklığı oluyor. Stack java yerine başka bir dosya ismi olacak şekilde ödevi yüklesem olur mu?

Hatalar: The method readCSV(String) is undefined for the type Stack
	     The import java.util.Stack conflicts with a type defined in the same file
             The type Stack is not generic; it cannot be parameterized with arguments <String>

**Comment by gusanmaz** [comment, marked as answer]
* ` ReadCSV` fonksiyonu `CSVReader.java` sinifinda olacak. Odev aciklamalari bu konuda oldukca net.

* `java.util.Stack` sinifindaki Stack'i kullanmayacaksiniz kendi Stack sinifinizi tanimlayacaksiniz.

Odev aciklamalarinda java.util.Stack kullanabilirsiniz demedim!

```
Bu programi kodlamak icin Stack veri yapisi tanimlayiniz. Stack veri yapisi projenizdeki Stack.java isimli dosyada tanimlanlamalidir.
```
